pipelines/src/main/java/au/org/ala/pipelines/interpreters/ALATemporalInterpreter.java

```diff
@@ -0,0 +1,105 @@
+package au.org.ala.pipelines.interpreters;
+
+import au.org.ala.pipelines.vocabulary.ALAOccurrenceIssue;
+import com.google.common.collect.Range;
+import org.apache.commons.lang3.StringUtils;
+import org.gbif.api.vocabulary.OccurrenceIssue;
+import org.gbif.common.parsers.core.OccurrenceParseResult;
+import org.gbif.common.parsers.core.ParseResult;
+import org.gbif.common.parsers.date.DateParsers;
+import org.gbif.common.parsers.date.TemporalAccessorUtils;
+import org.gbif.common.parsers.date.TemporalParser;
+import org.gbif.dwc.terms.DwcTerm;
+import org.gbif.pipelines.core.interpreters.core.TemporalInterpreter;
+import org.gbif.pipelines.io.avro.ExtendedRecord;
+import org.gbif.pipelines.io.avro.TemporalRecord;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.temporal.TemporalAccessor;
+
+import static org.gbif.pipelines.parsers.utils.ModelUtils.*;
+
+public class ALATemporalInterpreter{
+    protected static final LocalDate MIN_LOCAL_DATE = LocalDate.of(1600, 1, 1);
+
+    /**
+     * Inherit from GBIF interpretTemporal method.
+     * Add extra assertions
+     * @param er
+     * @param tr
+     */
+    public static void interpretTemporal(ExtendedRecord er, TemporalRecord tr) {
+        TemporalInterpreter.interpretTemporal(er,tr);
+        checkRecordDateQuality(er,tr);
+        checkDateIdentified(tr);
+        checkGeoreferencedDate(er,tr);
```

Comment:
It compares 'EventDate' and 'GeoreferencedDate'. Anyway, It needs to refer both LocationRecord and TemporalRecord. So I think putting this method into either LocationInterpreter or TemporalInterpreter , is OK. I tend to put into TemporalRecord, since interpreting EventDate is much more complicated.